spec/helpers/jsxc_helper_spec.rb

```diff
@@ -2,20 +2,20 @@
 
 describe JsxcHelper, :type => :helper do
   before do
-    AppConfig.chat.server.bosh.proxy = false
     AppConfig.chat.server.bosh.port = 1234
-    AppConfig.chat.server.bosh.bind = '/bind'
-    AppConfig.environment.url = "https://localhost/"
-    AppConfig.instance_variable_set(:@pod_uri, nil)
+    AppConfig.chat.server.bosh.bind = "/bind"
   end
 
   describe "#get_bosh_endpoint" do
     it "using http scheme and default values" do
+      AppConfig.chat.server.bosh.proxy = false
       expect(helper.get_bosh_endpoint).to include %Q(http://localhost:1234/bind)
     end
 
     it "using https scheme and no port" do
       AppConfig.chat.server.bosh.proxy = true
+      allow_any_instance_of(Configuration::Methods).to receive(:pod_uri)
+        .and_return(Addressable::URI.parse("https://localhost/"))
```

Comment:
`AppConfig` should be the only instance, so stubbing it directly should be fine.